charts/fybrik/templates/manager-deployment.yaml

```diff
@@ -68,6 +68,17 @@ spec:
           - containerPort: 8443
             name: https
         {{- end }}
+        {{ if .Values.storageManager.image }}
+        - name: storage-manager
+          image: {{ include "fybrik.image" ( tuple $ .Values.storageManager ) }}
+          imagePullPolicy: {{ .Values.manager.imagePullPolicy | default .Values.global.imagePullPolicy }}
+          ports:
+          - containerPort: {{ .Values.storageManager.serverPort }}
+            name: https
```

Comment:
why do you cal the port https? it can mislead if you don't use https. Why not something like `storage-manager` ?

Reply:
renamed to storage-manager